hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2966,6 +2967,20 @@ public String getRpcPort() {
   }
 
   @Override
+  public String getOmRatisRoles() {
+    List<ServiceInfo> serviceList = null;
+    int port = omNodeDetails.getRatisPort();
+    RaftPeer leaderId;
+    try {
+      leaderId = omRatisServer.getLeader();
```

Comment:
Should it handle the case when Ratis is not enabled in OM?

Reply:
I have added a check for Ratis!! 
thanks for the suggestion !!